TopAnalysis/src/FCNCTriLeptonCppWorker.cc

```diff
@@ -151,10 +148,15 @@ bool FCNCTriLeptonCppWorker::isNPElectron(const unsigned i) const {
   if ( pt < minElectronPt_ or std::abs(eta) > maxElectronEta_ ) return false;
   if ( std::abs(scEta) > 1.4442 and std::abs(scEta) < 1.566 ) return false;
   //nanoAOD object -> Electron_vidNestedWPBitmapSum16 (total 10 cuts in here by bitmap, each cut has 3 bit)
-  //int vidNestedWPBitmapSum16 = in_Electrons_vidNestedWPBitmapSum16->At(i);
-  //if ( ! ( vidNestedWPBitmapSum16 > ... ) ) return false;
+  const unsigned vidBitmap = in_Electrons_vidBitmap->At(i);
+  for ( unsigned i=0; i<10; ++i ) {
+    const unsigned vidBit = (vidBitmap & (0x7<<(3*i))); // 0x7 = 111 in the binary representation
+    const bool isLooseOnThisBit = vidBit >= 2; // 3 bits to store cuts. 0:fail, 1:veto, 2:loose, 3:medium, 4:tight
+    //if ( i != 5 and !isLooseOnThisBit or i == 5 and  isLooseOnThisBit ) return false;
+    if ( (i == 5) == (isLooseOnThisBit) ) return false;
```

Comment:
The condition will be (i == 2).
isocut is 3rd part from lower bits.

Reply:
Also, the isolation criteria is not inverted loose but "inverted tight" criteria. It means, isolation criteria should be fail, veto, loose and medium.

Reply:
fixed.